src/components/LandingPage/UploadForm/index.jsx

```diff
@@ -4,49 +4,65 @@ import { useNavigate } from "react-router-dom";
 import axios from "axios";
 import { IoIosAddCircle, IoIosCloseCircle } from "react-icons/io";
 
-import useUrlStore from "../../../store";
+import Message from "../../Message";
 import API from "../../../../config";
+import Loading from "../../shared/Loading";
+import { useAwsUrlStore, useYouTubeUrlStore } from "../../../store";
+import { QUALITY_MESSAGE, PLAYTIME_ALERT } from "../../../constants/message";
 
 function UploadForm() {
-  const [clicked, setClicked] = useState(false);
-  const [videos, setVideos] = useState({
-    mainVideo: "",
-    firstSubVideo: "",
-    lastSubVideo: "",
-  });
-
-  const { setVideoUrls } = useUrlStore();
-
   const navigate = useNavigate();
+  const [isClicked, setIsClicked] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+  const [serverMessage, setServerMessage] = useState("");
+  const { setVideoUrls, setAudioUrls } = useAwsUrlStore();
+  const { youtubeUrls, setYoutubeUrls } = useYouTubeUrlStore((state) => state);
 
   function handleClick() {
-    setClicked((prevState) => !prevState);
+    setIsClicked((prevState) => !prevState);
   }
 
   function handleChange(event) {
     const { name, value } = event.target;
 
-    setVideos((prevUrls) => ({
-      ...prevUrls,
-      [name]: value,
-    }));
+    setYoutubeUrls({ ...youtubeUrls, [name]: value });
   }
 
-  async function handleSubmit(event) {
+  async function requestUrls(event) {
     event.preventDefault();
+    setIsLoading(true);
 
     try {
-      const response = await axios.post(
-        API.VIDEOS,
-        { videoUrls: videos },
-        { withCredentials: true },
-      );
+      const response = await axios.post(API.CONTENTS, {
+        videoUrls: youtubeUrls,
+        isPermitted: false,
+      });
+
+      const { result } = response.data;
```

Comment:
아래 message도 같이 선언해줄 수 있을 것 같습니다!

```suggestion
      const { result, message } = response.data;
```

Reply:
클라이언트의 `CONTENTS` 요청이 성공할 경우 `message`는 담겨 있지 않아서 구조 분해할당시 에러가 날 수 있어서 분리를 해놓았었는데 저렇게 구조분해 할당하고 성공시에도 `message`를 전달하면 더 좋겠네요! 더 명시적인 것 같습니다!